### DISCUSSION TITLE
Package in Arch Linux's AUR

### DISCUSSION THREAD

**Discussion by nogweii** [zk-org/zk#45] [Share]
Hi! Love the tool, especially the LSP integration. With that, I've made and uploaded [this to the AUR](https://aur.archlinux.org/packages/zk/).

Thank you very much!

**Comment by mickael-menu** [comment]
Thanks @nogweii, this is great!

I'm not familiar with AUR, is it something that will need to be updated after each new release or it builds from the latest GitHub tag?

**Reply by nogweii** [reply]
Yeah, the AUR is a collection of [PKGBUILD](https://wiki.archlinux.org/title/PKGBUILD) files, and any associated supplementary files. The PKGBUILD is Arch Linux's way of specifying how to build a package from some upstream source (compile the source code, download a binary from this website, etc) and arrange the files into a directory to be compressed together as an Arch package. The tool managing that process is called [`makepkg`](https://wiki.archlinux.org/title/Makepkg).

PKGBUILD is comparable to RPM's spec files, or a bit like a Podfile (though that's stretching it a bit).

> is it something that will need to be updated after each new release?

Yeah, the PKGBUILD specifies the version directly and uses a git tag to ensure the right version is being compiled. I've subscribed to github notifications about version releases, we'll see if that's enough.
